lib/jnpr/junos/transport/tty.py

```diff
@@ -50,7 +50,7 @@ class Terminal(object):
         _re_pat_login,
         '(?P<passwd>assword:\s*$)',
         '(?P<badpasswd>ogin incorrect)',
-        '(?P<already_closed>session end at .*\n%)',
+        '(?P<netconf_closed>\<\!\-\- session end at .*\-\-\>\s*)',
```

Comment:
We only need to escape control characters, (+ ? . * ^ $ ( ) [ ] { } | \), So why did we opt to use escape for "!,<,>,-"